docs/docs/use_cases/question_answering/quickstart.mdx

```diff
@@ -75,20 +75,6 @@ We’ll use the following packages:
 %pip install --upgrade --quiet  langchain langchain-community langchainhub langchain-openai langchain-chroma bs4
 ```
 
-We need to set environment variable `OPENAI_API_KEY` for the embeddings model, which can be done
-directly or loaded from a `.env` file like so:
```

Comment:
`OpenAIEmbeddings` is used in this example so shouldn't there be a way to set the api key?

Reply:
> `OpenAIEmbeddings` is used in this example so shouldn't there be a way to set the api key?

Thanks for the reminder